### ISSUE TITLE
Problem with inheritance on Windows

### ISSUE THREAD

**Issue by maran** [go-qml/qml#77]
I'm having a problem making inherited components work on Windows. The use-case that works on Windows is the following.

The folder assets/qml has the following two files.

QmlApp.qml

``` qml
import QtQuick 2.0
import QtQuick.Controls 1.0;
import QtQuick.Layouts 1.0;

ApplicationWindow {
    minimumWidth: 500
    maximumWidth: 500
    maximumHeight: 400
    minimumHeight: 400
}
```

test_app.qml

``` qml
import QtQuick 2.0
import QtQuick.Controls 1.0;
import QtQuick.Layouts 1.0;
import Ethereum 1.0

QmlApp {
    minimumWidth: 350
    maximumWidth: 350
    maximumHeight: 80
    minimumHeight: 80

    title: "Test"
}
```

These are a bit simplified but it shouldn't matter for the problem I'm trying to describe.

When I use `loadFile` on OS X to open the test_app.qml it loads perfectly. However when I do the same on Windows I get an error.

```
2014/07/09 13:58:20 [GUI] assets/qml/test_app.qml:6 QmlApp is not a type
```

Is this expected behaviour? Is there a way for me to somehow tell where to look for parent components?

**Comment by niemeyer** [comment]
Can you please make sure you're using an up-to-date version of the qml package?  I recall we covered path issues on Windows in the mailing list, and this was fixed on e6047566.

**Comment by niemeyer** [comment]
Closing for lack of feedback.
